lib/github_changelog_generator/generator/section.rb

```diff
@@ -55,7 +55,7 @@ def get_string_for_issue(issue)
     end
 
     def issue_line_with_body(line, issue)
-      return issue["body"] if @body_only && issue["body"].present?
+      return issue["body"].gsub(/(\r\n|\r|\n)/, "\s\s\n") if @body_only && issue["body"].present?
```

Comment:
So, why do you want to `gsub` the whole "body"?

Wouldn't it easier to explain users, they need to put one extra break after the image. I believe it should represent the `issue["body"]` in the same way anywhere.